bin/helpers/sync/specsSummary.js

```diff
@@ -32,6 +33,24 @@ let printSpecsRunSummary = (data, machines) => {
     logger.info(`Total tests: ${summary.total}, passed: ${summary.passed}, failed: ${summary.failed}, skipped: ${summary.skipped}`);
     logger.info(`Done in ${data.duration/1000} seconds using ${machines} machines\n`);
 
+    if (customErrorsToPrint && customErrorsToPrint.length > 0) {
+      for (const error of customErrorsToPrint) {
+        switch(error.level) {
+          case 'info':
+            winstonlogger.info(error.message);
+            break;
+          case 'warn':
+            winstonlogger.warn(error.message);
+            break;
+          case 'error':
+            winstonlogger.error(error.message);
+            break;
+          default:
+            winstonlogger.warn(error.message);
```

Comment:
can remove
```
case 'warn':
            winstonlogger.warn(error.message);
            break;
```

It will be taken care of in default block

Reply:
Done